java/sage/Seeker.java

```diff
@@ -1241,6 +1241,17 @@ else if (autoImportAddedNetPaths.size() > 0)
       writeArchiveDirsProps();
     }
 
+    // Even though this is the only method that manipulates this variable, it is run on a thread
```

Comment:
I'm going to remove it. I know it can be hard to know when volatile is actually needed, but I tend to err on the side of caution unless there's a serious performance implication. In the later case I just try to find a different approach that doesn't have the potential problem. I'll just remove the code since I can't work out myself why this could ever be helpful.